docs/conf/portland/2025/cfp.rst

```diff
@@ -227,11 +249,20 @@ You'll be able to edit your proposal up until the submission deadline. Please be
 
 {% endif %}
 
-Write the Docs Atlantic
+Questions?
+----------
+
+If you have any questions, please email us at `{{email}} <mailto:{{email}}>`_ and let us know.
+
+{% if flagcfp %}
+
+Write the Docs Atlantic 
 -----------------------
 
 If you want to speak at Write the Docs but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, coming later in the year.
 
 This conference is entirely virtual, between the Central European Summer Time (CEST) and Eastern Daylight Time (EDT) time zones, so you can present or attend from almost anywhere.
 
 Talks for Write the Docs Atlantic are pre-recorded, with live Question and Answer sessions.
+
+{% endif %}
```

Comment:
I suspect we want to delete this whole section as in the blog post?

Reply:
Yea, makes sense.